mbslave/replication.py

```
@@ -238,44 +250,119 @@ def check_table_exists(db, schema, table):
+            schema, table = parse_name(config, name)
+            fulltable = fqn(schema, table)
+            if schema in ignored_schemas:
+                logger.info("Ignoring %s", name)
+                continue
+            if table in ignored_tables:
+                logger.info("Ignoring %s", name)
+                continue
+            if not check_table_exists(db, schema, table):
+                logger.info("Skipping %s (table %s does not exist)", name, fulltable)
+                continue
+            cursor.execute("SELECT 1 FROM %s LIMIT 1" % fulltable)
```

Ruff (0.14.10)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)